%YAML:1.0
DOF: 25
Head:
  NumberMotor: 2
	ModelMotor: AX12
	Motor1:
		ID: 1
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 2
		Init: 1024
		Min: 0
		Max: 2048
LeftArm:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 5
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 4
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 3
		Init: 1024
		Min: 0
		Max: 2048
LeftHip:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 11
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 10
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 9
		Init: 1024
		Min: 0
		Max: 2048
LeftKnee:
	NumberMotor: 1
	ModelMotor: MX64
	Motor1:
		ID: 12
		Init: 1024
		Min: 0
		Max: 2048
LeftFoot:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 15
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 14
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 13
		Init: 1024
		Min: 0
		Max: 2048
Waist:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 17
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 16
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 18
		Init: 1024
		Min: 0
		Max: 2048
RightArm:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 6
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 7
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 8
		Init: 1024
		Min: 0
		Max: 2048
RightHip:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 19
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 20
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 21
		Init: 1024
		Min: 0
		Max: 2048
RightKnee:
	Motor2: 1
		ID: 22
		Init: 1024
		Min: 0
		Max: 2048
RightFoot:
	NumberMotor: 3
	ModelMotor: MX64
	Motor1:
		ID: 23
		Init: 1024
		Min: 0
		Max: 2048
	Motor2:
		ID: 24
		Init: 1024
		Min: 0
		Max: 2048
	Motor3:
		ID: 25
		Init: 1024
		Min: 0
		Max: 2048
